### DISCUSSION TITLE
Keystore Management

### DISCUSSION THREAD

**Discussion by Prashantpv** [Opster/opensearch-k8s-operator#298] [Ideas]
Hi Folks,

I want to add certain secure settings via opensearch keystore. I couldn't find any relevant options for the same in operator. Currently I have modified the operator to execute some keystore command before opensearch-docker-entrypoint.sh execution. Please help me with correct way to adding the keystore with current operator capabilities

**Comment by swoehrl-mw** [comment]
Hi @Prashantpv. Currently it is not possible to configure the opensearch keystore via the operator. There is also an issue (#300) for this.

**Comment by lieberlois** [comment]
@Prashantpv Feel free to take a look at #314 , this aims to solve your problem 😄
